vsintegration/src/FSharp.Editor/Classification/ClassificationDefinitions.fs

```diff
@@ -41,6 +46,63 @@ module internal FSharpClassificationTypes =
         | SemanticClassificationType.Interface -> Interface
 
 module internal ClassificationDefinitions =
+
+    [<Export>]
+    type internal ThemeColors
+        [<ImportingConstructor>]
+        (
+            classificationformatMapService: IClassificationFormatMapService,
+            classificationTypeRegistry: IClassificationTypeRegistryService,
+            [<Import(typeof<SVsServiceProvider>)>] serviceProvider: IServiceProvider
+        ) =
+
+        let (| LighTheme | DarkTheme | UnknownTheme |) id =
+            if id = Guid("de3dbbcd-f642-433c-8353-8f1df4370aba") ||
```

Comment:
Can we shove these guids in named constants.  The OCD devil in me wants to read every digit of the guid.